api/src/test/java/io/opencensus/trace/StartSpanOptionsTest.java

```diff
@@ -30,64 +30,64 @@
 
   @Test
   public void defaultOptions() {
-    StartSpanOptions defaultOptions = new StartSpanOptions();
-    assertThat(defaultOptions.getSampler()).isNull();
-    assertThat(defaultOptions.getParentLinks().isEmpty()).isTrue();
-    assertThat(defaultOptions.getRecordEvents()).isNull();
+    assertThat(StartSpanOptions.DEFAULT.getSampler()).isNull();
+    assertThat(StartSpanOptions.DEFAULT.getParentLinks().isEmpty()).isTrue();
+    assertThat(StartSpanOptions.DEFAULT.getRecordEvents()).isNull();
```

Comment:
I think it would be better to test the API by either exposing the DEFAULT constant or testing `StartSpanOptions.builder().build()` in this test.